app/assets/javascripts/miq_application.js

```diff
@@ -737,7 +737,12 @@ function miqAjax(url, serialize_fields, options) {
     complete: true,
   };
 
-  miqJqueryRequest(url, _.extend(defaultOptions, options || {}, { data: data }));
+  miqJqueryRequest(url, _.extend(defaultOptions, options || {}, { data: data }))
+    .catch(function(err){
+      add_flash(__("Error requesting data from server"), 'error');
+      console.log(err);
+      return Promise.reject(err)
+  });
```

Comment:
tiny detail but the `});` should be aligned under `.catch`, not under `miqJqueryRequest`..